lib/properties.ex

```diff
@@ -283,4 +284,25 @@ defmodule PropCheck.Properties do
       {:ok, {_, [{:abstract_code, {_, ac}}]}} = :beam_lib.chunks(beam, [:abstract_code])
       ac |> Enum.map(&:erl_pp.form/1) |> List.flatten |> IO.puts
   end
+
+  @doc """
+  Defines a not implemented property.
```

Comment:
Suggestion: `Defines a not yet implemented property.`

Reply:
I was wondering about the same thing :) I took it from ExUnit.Case:

>   Defines a not implemented test with a string.

But I will apply your suggestion.